content/blog/2017/08/2017-08-07-intro-to-plugin-development.adoc

```diff
@@ -0,0 +1,68 @@
+---
+layout: post
+title: "Plugin Development Tutorials, Videos, and More"
+tags:
+- plugins
+author: markewaite
+:experimental: true
+---
+
+While developing the "link:https://www.cloudbees.com/intro-plugin-development-workshop[Intro to Plugin Development]"
```

Comment:
This blog post should start with a guest post preamble, refer to some recent guest posts for examples